### DISCUSSION TITLE
Welcome to Jermyr Discussions!

### DISCUSSION THREAD

**Discussion by Jrjrjr23** [Jrjrjr23/Jermyr#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nicoleash333** [comment]
Hey

**Reply by Jrjrjr23** [reply]
Yeah

**Comment by Jrjrjr23** [comment]
Wyd

**Comment by Jrjrjr23** [comment]
I hope you're not on any dating sites or anything I have no dating sites. I never wanted them. That is the gospel truth right there please don't doubt that that is so true. I literally did not want any dating apps. I was ****against**** it for you and me.
